patches/tModLoader/Terraria/ModLoader/ItemLoader.cs

```diff
@@ -26,6 +26,7 @@ namespace Terraria.ModLoader;
 public static class ItemLoader
 {
 	public static int ItemCount { get; private set; } = ItemID.Count;
+	public static int UseStyleCount { get; private set; } = ItemUseStyleID.RaiseLamp + 1;
```

Comment:
I'd rather see a `Count` field added to `ItemUseStyleID`. It's more correct and we're more likely to pick it up in patch reviewer.